README.md

```diff
@@ -5,21 +5,76 @@
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+The current release of `cri-containerd` is `v0.1.0-alpha.1`. This release is for
+use with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones)
+for information about current and future milestones.
+## Social
+This is a project within the Kubernetes SIG named `sig-node`.
+For more information about `sig-node` and the `cri-containerd` project:
+* [sig-node community site](https://github.com/kubernetes/community/tree/master/sig-node).
+* Catch us on Kubernetes (kubernetes.slack.com) in the `#sig-node` channel.
+* Mailing List: https://groups.google.com/forum/#!forum/kubernetes-sig-node
+## Getting started
+### Binary Dependencies
+The alpha release of cri-containerd has been tested on [containerd](https://github.com/containerd/containerd) v0.2.3-1098-g8ed1e24, [runc](https://github.com/opencontainers/runc) v1.0.0-rc3, and current releases of [CNI](https://github.com/containernetworking/cni).
+### Build cri-containerd
+1. Install and setup a go1.8.x development environment.
+1. Make a local clone of this repository.
+1. From your github.com/kubernetes-incubator/cri-containerd/ path (Note: run these commands as root):
+```
+make install.tools
+make
+make install
+```
+### Running with cri-tools
+Another Kubernetes incubator project called cri-tools includes programs for
+exercising CRI implementations such as `cri-containerd`. These programs include:
+* `critest`: a testing tool for running CRI verification tests.
+* `crictl`: a command line interface (CLI) for managing pods and containers.
+
+1. Install cri-tools
+```
+go get github.com/kubernetes-incubator/cri-tools/cmd/critest
+go get github.com/kubernetes-incubator/cri-tools/cmd/crictl
+```
+2. Run `containerd` in a first terminal
+```
+containerd -l debug
+```
+3. Run `cri-containerd` in a second terminal
+```
+cri-containerd -v 4 --alsologtostderr
+```
+4. Run cri-tool's verification tests against cri-containerd in a third terminal
+```
+critest --runtime-endpoint=/var/run/cri-containerd.sock validation
+```
+You can also test out cri-containerd via [crictl](https://github.com/kubernetes-incubator/cri-tools#crictl).
+### Running with Kubernetes
+If you already have a working development environment for Kubernetes, you can
+try cri-containerd in a local cluster:
+
+1. Start `containerd` in a first terminal
+```
+containerd -l debug
+```
+2. Start `cri-containerd` in a second terminal
+```
+cri-containerd -v 4 --alsologtostderr
+```
+3. From the kubernetes project directory startup a local cluster using cri-containerd for the runtime implementation
+```
+KUBERNETES_PROVIDER=local CONTAINER_RUNTIME=remote CONTAINER_RUNTIME_ENDPOINT='/var/run/cri-containerd.sock' ./hack/local-up-cluster.sh
```

Comment:
but it's not a developer.. and local is need to run locally.. are you saying change to show multiple options if they have a GCE account?